contracts/PFA.sol

```diff
@@ -52,13 +52,9 @@ abstract contract PFA is IPFA, LimitedOwnable {
 
     /// @notice Sets the price per access in wei for content backed
     /// by this contract.
-    function setPricePerAccess(uint256 pricePerAccess_)
-        public
-        override
-        nonReentrant
-        onlyOwner
-        afterInit
-    {
+    function setPricePerAccess(
```

Comment:
@DeusExM4ch1n4 Can you identify what's causing the formatting issue and restore? For audit purposes / internal consistency would like to keep only the critical diffs.

Reply:
This change comes from the standard Prettier Solidity formatter
https://github.com/prettier-solidity/prettier-plugin-solidity

Reply:
I will revert the modifications to the stylization.